src/routes/edit/share-options/exports/frameworks/angular/v10/utils.ts

```diff
@@ -30,7 +30,7 @@ export const jsonToAngularImports = (json: any) => {
 
 	for (const component of Object.values(allComponents)) {
 		if (json.type === component.componentInfo.type) {
-			addIfNotExist(imports, component.componentInfo.codeExport.angular?.imports);
+			addIfNotExist(imports, component.componentInfo.codeExport.angular?.v10.imports);
```

Comment:
`component.componentInfo.codeExport.angular?` appears all over the file and then takes v10 from it.

Would it make sense (if it's the same object) to assign it to a variable and then use that variable? (in latest as well)

This way we only need to assign that variable to `v*` when  moving/upgrading instead of search&replace.